pkg/agent/attestor/node/node.go

```diff
@@ -81,10 +83,19 @@ func (a *attestor) Attest(ctx context.Context) (res *AttestationResult, err erro
 		return nil, err
 	}
 
+	reattestable, err := storage.ReadReattestable(a.c.ReattestablePath)
+	if err != nil {
+		return nil, err
+	}
+
 	switch {
-	case svid == nil:
-		log.Info("SVID is not found. Starting node attestation")
-		svid, bundle, err = a.newSVID(ctx, key, bundle)
+	case svid == nil || reattestable:
```

Comment:
Attestation is called unconditionally on startup. With this change, won't the agent re-attest on startup whether or not the SVID is still valid? Has this behavior change been discussed somewhere?

Reply:
I ended up creating two functions. `Attest()` which just blindly attests and `AttestIfNoSVID()` which attests only no existing svid is found.